tests/snapshot_test.py

```diff
@@ -103,12 +95,8 @@ def test_derived_array():
     f['vr']
     assert 'vr' in f.keys()
 
-    try:
+    with pytest.raises(ValueError):
```

Comment:
The following line only raises a `ValueError`, and neither a `RuntimeError` or a `TypeError`, so only catch the former.